tools/topology/topology2/include/pipelines/cavs/dai-copier-asrc-capture.conf

```diff
@@ -0,0 +1,72 @@
+#
+# CAVS ASRC capture pipeline
+#
+# A simple asrc pipeline. All attributes defined herein are namespaced
+# by alsatplg to "Object.Pipeline.dai-copier-asrc-capture.N.attribute_name".
+#
+# Usage:dai-copier-asrc-capturepipeline object can be instantiated as:
+#
+# Object.Pipeline.dai-copier-asrc-capture."N" {
+# 	format		"s16le"
+# 	period		1000
+# 	time_domain	"timer"
+# 	channels	2
+# 	rate		48000
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+<include/common/audio_format.conf>
+<include/components/dai-copier.conf>
+<include/components/asrc.conf>
+<include/components/pipeline.conf>
+
+Class.Pipeline."dai-copier-asrc-capture" {
+
+	DefineAttribute."index" {}
+
+	<include/pipelines/pipeline-common.conf>
+
+	attributes {
+		!constructor [
+			"index"
+		]
+
+		!immutable [
+			"direction"
+		]
+
+		unique	"instance"
+	}
+
+	Object.Widget {
+		dai-copier."1" {
+			type dai_out
+			<include/components/asrc_format_sxx_to_s32_convert.conf>
+			num_output_pins 1
+		}
+
+		asrc."1" {
+			asynchronous_mode	1
+			operation_mode		1
```

Comment:
Those two attributes could be transformed to string type to make it human readable.